content/v2.1/en/quick-start/wordpress-deployment.md

```diff
@@ -1,153 +1,149 @@
 
+- Key: `MYSQL_ROOT_PASSWORD`
+- Value: `123456`
 
-- Key: MYSQL_ROOT_PASSWORD
-- Value: 123456
+![Create MySQL Secret](https://pek3b.qingstor.com/kubesphere-docs/png/20200105182805.png)
 
-![](https://pek3b.qingstor.com/kubesphere-docs/png/20200105182805.png)
+#### Create a WordPress Secret
 
-### Create a WordPress Secret
+Same steps as above, create a WordPress secret `wordpress-secret` with `Key: WORDPRESS_DB_PASSWORD` and `Data: 123456`.
 
-Same steps as above, create a WordPress Secret `Wordpress-secret` with `Key : WORDPRESS_DB_PASSWORD` and `Data : 123456`.
+![Create WordPress Secret](https://pek3b.qingstor.com/kubesphere-docs/png/20200105183314.png)
 
-![](https://pek3b.qingstor.com/kubesphere-docs/png/20200105183314.png)
+### Step 2: Create a Volume
 
-## Create a Volume
+Choose **Volumes** and click **Create**, name it `wordpress-pvc`, click **Next** to Volume Settings where you need to choose an available `Storage Class`, `ReadWriteOnce` of access mode and 10 G of storage size. Click `Next` to Advanced Settings. No configuration is for this page, so click **Create** to finish volume creation.
 
-Choose **Volumes** and click **Create**, name it `wordpress-pvc`, then click **Next** to skip Volume Settings and Advanced Settings, click **Create** to finish volume creation.
+![Create Volume](https://pek3b.qingstor.com/kubesphere-docs/png/20200106000543.png)
 
-![](https://pek3b.qingstor.com/kubesphere-docs/png/20200106000543.png)
+### Step 3: Create an Application
 
-## Create an Application
+#### Add MySQL back-end component
 
-### Add MySQL back-end component
+In this step, we will choose the way of composing app to create a complete microservice app.
 
-In this section, we will choose composing app to create a complete microservice app.
+3.1. Select **Application Workloads → Applications → Deploy New Application**, and choose **Composing App**.
 
-1. Select **Application Workloads → Application → Deploy New Application**, and choose **Composing App**.
+![New Application](https://pek3b.qingstor.com/kubesphere-docs/png/20200106000851.png)
 
-![](https://pek3b.qingstor.com/kubesphere-docs/png/20200106000851.png)
+3.2. Fill in the pop-up table as follows:
 
-2. Fill in the pop-up table as follows:
-
-
-- Application Name: wordpress
+- Application Name: `wordpress`
 - Then click **Add Component**
-- Name: mysql
-- Component Version: v1
+- Name: `mysql`
+- Component Version: `v1`
 - Workload Type: Stateful service (StatefulSet)
 
-![](https://pek3b.qingstor.com/kubesphere-docs/png/20200106001425.png)
+![Compose Application](https://pek3b.qingstor.com/kubesphere-docs/png/20200106001425.png)
 
-3. Click **Add Container Image**, enter `mysql:5.6` into Image, press the return key and click `Use Default Ports`.
+3.3. Click **Add Container Image**, input `mysql:5.6` in the Image edit box, press the return key and click `Use Default Ports`.
```

Comment:
I think `enter` is also correct, this usage I always see in [Azure docs](https://docs.microsoft.com/en-us/azure/devops/test/load-test/get-started-simple-cloud-load-test?view=azure-devops#run-a-url-based-load-test).